conda-envs/windows-environment-test-py38.yml

```diff
@@ -0,0 +1,37 @@
+name: pymc3-test-py38
+channels:
+- conda-forge
+- defaults
+dependencies:
+ # base dependencies (see install guide for Windows)
+- aesara>=2.1.0
+- arviz>=0.11.2
+- cachetools>=4.2.1
+- cloudpickle
+- fastprogress>=0.2.0
+- h5py>=2.7
+- libpython
+- mkl-service
+- m2w64-toolchain
+- numba
```

Comment:
This is the hack to get compiler dependencies installed correctly. Can we remove it already, or is the Aesara recipe still broken?